docs/design/schedule-customisation-design.md

```diff
@@ -0,0 +1,29 @@
+# Customize Scheduling Algorithms per Queue
```

Comment:
please add the interaction with existing features.

Reply:
Updated [Link](https://github.com/volcano-sh/volcano/pull/1143/files#diff-9d6403c433ff965dac666d298542c7a98df50791bd61ba2ab3340ab75ad49db5R27)